actions.yaml

```diff
@@ -0,0 +1,2 @@
+run-migration:
+  description: Run a command to create SQL schemas and apply migration plans.
```

Comment:
In the future, please make the PR's simplers. Don't mix in new features with refactoring

Reply:
I'll try to keep the changes separate in future PRs. Adding the action was a part of addressing review comments. Thanks